src/routes/utils.ts

```diff
@@ -0,0 +1,26 @@
+import { up } from '../db/initialize';
+import { readFile } from 'fs/promises';
+import Router from 'koa-router';
+import { Utils } from '../utils/utils';
+
+export async function logsRoute(ctx: Router.RouterContext) {
+  try {
+    ctx.body = await readFile('./logs', 'utf-8');
+    return;
+  } catch (error) {
+    console.error({ error });
+  }
+}
+
+export async function resetRoute(ctx: Router.RouterContext) {
+  try {
+    ctx.network.blocks = 1;
+    ctx.network.height = 0;
+    ctx.network.current = Utils.randomID(64);
```

Comment:
Hey @kouraf, I think you need to persist this block to DB. To prevent the issue fixed here - https://github.com/textury/arlocal/pull/101 from occurring again.